tests/jballerina-debugger-integration-test/src/main/java/org/ballerinalang/debugger/test/utils/BreakpointEventListener.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinalang.debugger.test.utils;
+
+import org.ballerinalang.debugger.test.utils.client.DAPClientConnector;
+import org.eclipse.lsp4j.debug.Breakpoint;
+import org.eclipse.lsp4j.debug.BreakpointEventArguments;
+import org.eclipse.lsp4j.debug.BreakpointEventArgumentsReason;
+
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.TimerTask;
+import java.util.concurrent.ConcurrentLinkedQueue;
+
+/**
+ * Timer Task implementation to capture breakpoint events.
+ */
```

Comment:
Shall we add `@since ` annotation here?

Reply:
@lnash94 I doubt whether we need to add it as this is a part of the debugger integration test suite and not a debug server API class. WDYT?

Reply:
Yeah sure we can keep this, I have mistakenly taken this as implementation. :)